WcaOnRails/app/controllers/api/v0/competitions_controller.rb

```diff
@@ -29,7 +29,34 @@ def schedule
 
   def results
     competition = competition_from_params
-    render json: competition.results
+    render json: competition
```

Comment:
Nit: this should actually stay as `competition.results` (that's why Travis is failing by the way ^^).